### PR TITLE
Add aria and focus management to loading indicator

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [202](https://github.com/department-of-veterans-affairs/sunsets-team/issues/202)

In the above issue, we got some feedback that there was no audio indication for the loading spinner.

This updates the loading indicator to have indeterminate `progressbar` aria attributes, which seemed most appropriate (based on [this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role)).

I put the focus changes behind a prop so it doesn't disrupt anyone else.

### CODE REVIEW

**Comment by jbalboni** [comment]
@bshyong and @crwallace, is this a reasonable solution? If we want to revise it later or keep these changes out of the global component, I can do that, too.

**Review by molindoz — APPROVED** [review]
LGTM!

File: src/js/common/components/LoadingIndicator.jsx [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+            aria-valuetext={message || screenReaderMessage}
+            tabIndex="0"></div>
+        {message}
+      </div>
+    );
+  }
 }
+
+LoadingIndicator.propTypes = {
+  message: React.PropTypes.string,
```
Comment (crwallace) [inline src/js/common/components/LoadingIndicator.jsx]: @jbalboni -- Can the `message` and the `screenReaderMessage` say the same thing? It is important to keep the experience consistent for sighted and non-sighted users. and the we can may the `message` `propType` required.

Comment (jbalboni) [inline src/js/common/components/LoadingIndicator.jsx]: @crwallace Works for me!

**Pull request merged**